compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -803,6 +829,30 @@ class PowTestGraph final : public SimpleTestGraph
   luci::CircleConst *_const = nullptr;
 };
 
+class ResizeBilinearTestGraph final : public luci::test::TestIOGraph
+{
+public:
+  void init(void)
+  {
+    TestIOGraph::init({1, 4, 4, 1}, {1, 8, 8, 1});
+
+    _size = create_const<Type::S32, int32_t>(g(), {2}, {8, 8});// TODO : check
+    _resizebilinear = g()->nodes()->create<luci::CircleResizeBilinear>();
+    {
+      _resizebilinear->input(input());
```

Comment:
Maybe size is missing?

Reply:
@jinevening 
It still throws the same exception 😢